.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,16 +16,13 @@
 
 If you need help, consider asking for advice on the #hackers-new channel on [Discord].
 
-**Note**: The Flutter team is currently trialing the use of [Gemini Code Assist for GitHub](https://developers.google.com/gemini-code-assist/docs/review-github-code). Comments from the `gemini-code-assist` bot should not be taken as authoritative feedback from the Flutter team. If you find its comments useful you can update your code accordingly, but if you are unsure or disagree with the feedback, please feel free to wait for a Flutter team member's review for guidance on which automated comments should be addressed.
-
 [^1]: Regular contributors who have demonstrated familiarity with the repository guidelines only need to comment if the PR is not auto-exempted by repo tooling.
 
 <!-- Links -->
 [Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
 [Tree Hygiene]: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md
 [relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
 [the auto-formatter]: https://github.com/flutter/packages/blob/main/script/tool/README.md#format-code
-[CLA]: https://cla.developers.google.com/
 [Discord]: https://github.com/flutter/flutter/blob/master/docs/contributing/Chat.md
 [linked to at least one issue that this PR fixes]: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md#overview
 [pub versioning philosophy]: https://dart.dev/tools/pub/versioning
```

Comment:
maybe we should create our own template with rules?

Reply:
I think that this is fine. We should continue original guidelines.